### PR TITLE
testcases: add ltp-smoketest

### PR DESCRIPTION
**Pull request by nareshkamboju**
LTP new release 20200930 introduced smoketest. 

ref:
https://github.com/linux-test-project/ltp/commit/0d732deeb7d3b0ed4bca06696738ad0678486742

Signed-off-by: Naresh Kamboju <naresh.kamboju@linaro.org>

### CODE REVIEW

**Comment by nareshkamboju** [comment]
@roxell If any pipeline using test case name as ltp-quickhit.yaml should be changed to ltp-smoketest.yaml
@mrchapp this patch should only be merged when LTP upgrade to 20200930 in new rootfs.

**Comment by roxell** [comment]
We can't just delete ltp-quickhit from testcases, if someone uses an LTP release that still have that he should be able to use it.

If we really want to remove it, I think we have to wait Xweeks and then remove it.

File: testplans/lkft-sanity/ltp-quickhit.yaml [hunk 1]
```diff
@@ -1 +0,0 @@
-../../testcases/ltp-quickhit.yaml
```
Comment (roxell) [inline testplans/lkft-sanity/ltp-quickhit.yaml:1]: Don't remove this from here either right now.
Others depends on it.
That may be wrong but that is how it is right now sadly.
Until that is fixed we have to leave this file here.
We will still be able to move over to ltp-smoketest for our new rootfs.

Comment (nareshkamboju) [reply testplans/lkft-sanity/ltp-quickhit.yaml]: @roxell Thanks for the review.
Your review comments have been addressed.

**Comment by roxell** [comment]
LGTM

**Pull request merged**